### PR TITLE
api: Implement POST /default_streams api endpoint.

### PR DESCRIPTION
**Pull request by akashaviator**
This implements the POST /default_streams api endpoint.

### CODE REVIEW

**Comment by akashaviator** [comment]
@timabbott Can you have a look ?

File: zulip/zulip/__init__.py [hunk 1]
```diff
@@ -1081,6 +1081,21 @@ def delete_stream(self, stream_id):
...
+        '''
+            Example usage:
+
+            >>> client.add_default_stream("Verona")
+            {'result': 'success', 'msg': ''}
+        '''
+        return self.call_endpoint(
+            url='default_streams',
+            method='POST',
+            request={'stream_name': stream_name},
```
Comment (timabbott) [inline zulip/zulip/__init__.py:1096]: I guess we can switch this to stream_id

**Comment by akashaviator** [comment]
@timabbott I've updated the PR.

**Comment by timabbott** [comment]
Looks great, merged, thanks @akashaviator!

**Pull request merged**